plugins/woocommerce-admin/client/index.js

```diff
@@ -93,7 +93,13 @@ if ( appRoot ) {
 		window.wcAdminFeatures &&
 		window.wcAdminFeatures[ 'launch-your-store' ] === true
 	) {
-		registerSiteVisibilitySlotFill();
+		const urlParams = new URLSearchParams( window.location.search );
+		if (
+			urlParams.get( 'tab' ) === 'general' ||
+			! urlParams.get( 'tab' )
```

Comment:
This feels like the wrong place to put this logic. I'm not entirely sure why Site Visibility is getting rendered in the tax bracket (I can't reproduce), but it seems more logical to make sure the root div `#wc_settings_general_site_visibility_slotfill` does not get rendered on any page except the general tab of Settings. I think this logic should be in one place.

Reply:
@psealock Sounds good. I'll look into a different approach.

Reply:
> I'm not seeing this. I think the event would be fired from PHP via wc_admin_record_tracks_event and not seen on the Network tab, right?

You can see it in the network tab after clicking `Save changes`. 

> This feels like the wrong place to put this logic

Removed it in https://github.com/woocommerce/woocommerce/pull/46078/commits/2426f7ef6deb29539cb60cedd558c80441aa6c02

This should be fixed in https://github.com/woocommerce/woocommerce/pull/46140 instead